### ISSUE TITLE
[dev.icinga.com #10369] Accessibility: Focus not visible and lost after refresh

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#2087]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/10369

**Created by tgelf on 2015-10-15 09:53:37 +00:00**

Assignee: _elippmann_
Status: _Resolved_  (closed on _2016-01-25 06:36:26 +00:00_)
Target Version: _2.2.0_
Last Update: _2016-01-25 06:36:26 +00:00 (in Redmine)_

---

It seems that we messed up our styles showing the current focus:

-   active form buttons show now focus information
-   links are showing just an outline again - is this by intention?
-   menu does not allow to distinct between selected and focussed element

Also, is it possible that focus is no longer restored on refresh?

Cheers,
Thomas
